model/registry/geo.yaml

```diff
@@ -0,0 +1,113 @@
+        type:
+          members:
+            - id: 'af'
+              stability: experimental
+              value: 'AF'
+              brief: 'Africa'
+            - id: 'an'
+              stability: experimental
+              value: 'AN'
+              brief: 'Antarctica'
+            - id: 'as'
+              stability: experimental
+              value: 'AS'
+              brief: 'Asia'
+            - id: 'eu'
+              stability: experimental
+              value: 'EU'
+              brief: 'Europe'
+            - id: 'na'
+              stability: experimental
+              value: 'NA'
+              brief: 'North America'
+            - id: 'oc'
+              stability: experimental
+              value: 'OC'
+              brief: 'Oceania'
+            - id: 'sa'
+              stability: experimental
+              value: 'SA'
+              brief: 'South America'
+      - id: continent.name
+        stability: experimental
+        type: string
+        brief: >
+          Name of the continent.
+        examples: [ 'North America', 'Europe' ]
+      - id: country.iso_code
+        stability: experimental
+        type: string
+        brief: >
+          Two-letter ISO Country Code ([ISO 3166-1 alpha2](https://en.wikipedia.org/wiki/ISO_3166-1#Codes)).
+        examples: [ 'CA' ]
+      - id: country.name
+        stability: experimental
+        type: string
+        brief: >
+          Country name.
+        examples: [ 'Canada' ]
+      - id: location.lon
+        stability: experimental
+        type: double
+        brief: >
+          Longitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ -73.614830 ]
+      - id: location.lat
+        stability: experimental
+        type: double
+        brief: >
+          Latitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ 45.505918 ]
+      - id: name
+        stability: experimental
+        type: string
+        brief: >
+          User-defined description of a location.
+        note: >
+          User-defined description of a location, at the level of granularity they care about.
+          Could be the name of their data centers, the floor number, if this describes a local physical entity, city names.
+          Not typically used in automated geolocation.
+        examples: [ 'boston-dc' ]
+      - id: postal_code
+        stability: experimental
+        type: string
+        brief: >
+          Postal code associated with the location.
+          Values appropriate for this field may also be known as a postcode or ZIP code and will vary widely from country to country.
+        examples: [ '94040' ]
+      - id: region.iso_code
+        stability: experimental
+        type: string
+        brief: >
+          Region ISO code ([ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2)).
+        examples: [ 'CA-QC' ]
+      - id: region.name
```

Comment:
From what I found, ECS added those because the [geoip processor collects](https://www.elastic.co/guide/en/elasticsearch/plugins/current/using-ingest-geoip.html#using-ingest-geoip) both: https://github.com/elastic/ecs/issues/177.

I think this is very similar to https://github.com/open-telemetry/semantic-conventions/pull/1116#discussion_r1668319586: information stored in both attributes is definitely redundant with the benefit of fixed values ([ISO 3166-2 code) vs. not needing to parse; and with having both, implementation can just choose.

Reply:
same here, let's not add attributes that we don't have a use-case for yet. Especially if they are controvercial.

Reply:
These are used by the collector processor: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33451/files#diff-0e91371ea4833d69a2fe3ac9e0ccab8db487e56af914d1717b0763a973eaffd4R92-R93

Reply:
does it need to record both? would users appreciate having both and paying more for the same data?

That's exactly the reason I'm pushing back on adding duplicative attributes - when people see a list of attributes, they populate them all. 

Let's pick one and add it. Collector process should remove the other one unless some good reason to populate both.

Suggesting iso_code since it's a standard. Backends can maintain a mapping to human-friendly names for all continents.

Reply:
> does it need to record both? would users appreciate having both and paying more for the same data?

That's a good point. The collector tends to disable metrics that can be derived by others in its default configurations, the same could be applied to attributes. The user would need to explicitly change the processor configuration to populate this attribute `region.name::enabled: true`.

Nonetheless, I support only including the `iso_code` attribute for this initial PR and leave the human-friendly name for a future PR or when we have an explicit use case. If needed, we could provide a `transformer` processor configuration to do a sample mapping.

Reply:
Thanks for the input. Removed `region.name`.